<!DOCTYPE html>
<html><meta http-equiv="Content-Type" content="text/html" charset="UTF-8" />
<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />

<title>Chromatin looping links target genes with genetic risk loci for dermatological traits</title>


  <link rel="shortcut icon" href="images/favicon.png" type="image/png"
/>


  <meta name="description" content="Chromatin looping links target genes with genetic risk loci for dermatological traits"
/>

  <meta name="keywords" content="HiChIP, Chenfu, Chromatin interactions, Chenfu Shi, Epigenomics, Bioinformatics"
/>
<meta name="referrer" content="no-referrer-when-downgrade" />

<meta name="HandheldFriendly" content="True" />
<meta name="MobileOptimized" content="320" />
<meta name="viewport" content="width=device-width, initial-scale=1.0" />

<link rel="stylesheet" href="css/screen.css" />
<link
  href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300,300italic,700"
  rel="stylesheet"
  type="text/css"
/>
<link
  href="https://fonts.googleapis.com/css?family=Oswald:400,300,700|Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800|Roboto+Slab:400,100,300,700"
  rel="stylesheet"
  type="text/css"
/>
<link
  rel="stylesheet"
  href="font-awesome/css/font-awesome.min.css"
/>

<meta property="og:title" content="" />
<meta property="og:description" content="Chromatin looping links target genes with genetic risk loci for dermatological traits" />
<meta property="og:type" content="website" />
<meta property="og:url" content="/" />
<meta property="og:image" content="images/figure_1.png"/>
<meta property="og:site_name" content="Chromatin looping links target genes with genetic risk loci for dermatological traits" />

<meta name="twitter:card" content="summary_large_image"/>
<meta name="twitter:image" content="images/figure_1.png"/>

<meta name="twitter:title" content=""/>
<meta name="twitter:description" content="Chromatin looping links target genes with genetic risk loci for dermatological traits"/>

<meta itemprop="name" content="">
<meta itemprop="description" content="Chromatin looping links target genes with genetic risk loci for dermatological traits">










<link rel="stylesheet" href='css/custom.css'>





<link
  rel="apple-touch-icon"
  sizes="180x180"
  href="images/apple-touch-icon.png"
/>
<link
  rel="icon"
  type="image/png"
  sizes="32x32"
  href="images/favicon-32x32.png"
/>
<link
  rel="icon"
  type="image/png"
  sizes="16x16"
  href="images/favicon-16x16.png"
/>
<body>


<header id="site-head" >
    <div class="vertical">
        <div id="site-head-content" class="inner">

            <h1 class="blog-title">Chromatin looping links target genes with genetic risk loci for dermatological traits</h1>
            

            <p>Chenfu Shi<sup>1</sup>, Helen Ray-Jones<sup>1,2</sup>, James Ding<sup>1</sup>, Kate Duffus<sup>1</sup>, Yao Fu<sup>3</sup>, Vasanthi Priyadarshini Gaddi<sup>1</sup>, Oliver Gough<sup>1</sup>, Jenny Hankinson<sup>4</sup>, Paul Martin<sup>1,5</sup>, Amanda McGovern<sup>1</sup>, Annie Yarwood<sup>1,2</sup>, Patrick Gaffney<sup>3</sup>, Steve Eyre<sup>1</sup>, Magnus Rattray<sup>7</sup>, Richard Warren<sup>2</sup> and Gisela Orozco<sup>1,6</sup>.</p>
<ol>
<li>Centre for Genetics and Genomics Versus Arthritis. Division of Musculoskeletal and Dermatological Sciences, School of Biological Sciences, Faculty of Biology, Medicine and Health, The University of Manchester, UK</li>
<li>Dermatology Centre, Salford Royal NHS Foundation Trust, Manchester NIHR Biomedical Research Centre, Manchester Academic Health Science Centre, Manchester, UK</li>
<li>Genes and Human Disease Research Program, Oklahoma Medical Research Foundation, Oklahoma City, 73104, OK, USA</li>
<li>Division of Infection, Immunity and Respiratory Medicine, School of Biological Sciences, University of Manchester, Manchester, UK</li>
<li>The Lydia Becker Institute of Immunology and Inflammation, University of Manchester, Manchester, UK</li>
<li>NIHR Manchester Biomedical Research Centre, Manchester University NHS Foundation Trust, Manchester Academic Health Science Centre, Manchester, UK</li>
<li>Division of Informatics, Imaging and Data Sciences, Faculty of Biology, Medicine and Health, University of Manchester, UK</li>
</ol>


            
                 <a class='btn site-menu' data-title-anchor='introduction'>Introduction</a>
            
                 <a class='btn site-menu' data-title-anchor='this-project'>This project</a>
            
                 <a class='btn site-menu' data-title-anchor='pathways'>Pathways</a>
            
                 <a class='btn site-menu' data-title-anchor='ets1'>ETS1</a>
            
                 <a class='btn site-menu' data-title-anchor='satb1'>SATB1</a>
            
                 <a class='btn site-menu' data-title-anchor='irf4'>IRF4</a>
            
                 <a class='btn site-menu' data-title-anchor='grhl3'>GRHL3</a>
            
                 <a class='btn site-menu' data-title-anchor='foxo1'>FOXO1</a>
            
                 <a class='btn site-menu' data-title-anchor='conclusion'>Conclusion</a>
            
                 <a class='btn site-menu' data-title-anchor='contact'>Contact</a>
            
            <i id='header-arrow' class="fa fa-angle-down"></i>
        </div>
    </div>
</header>
<main class="content" role="main">

    <div class='fixed-nav'>
    </div>
    
        <div class='post-holder'>
            <article id='introduction' class='post first'>
                <header class="post-header">
                    <h2 class="post-title" style="display:none">Introduction</h2>
                </header>
                <section class="post-content">
                    <h2 id="welcome">Welcome</h2>
<p>This website summarizes the main results from our preprint available on <a href="https://www.biorxiv.org/content/10.1101/2020.03.05.973271v2">BioRxiv</a>.</p>
<p>It was written by Chenfu Shi as an addition to the poster presented at the Wellcome Epigenomics of Common Diseases 2020 online conference to make our results clearer and more accessible.</p>
<h5 id="functional-genomics-of-common-diseases">Functional genomics of common diseases</h5>
<p>Genome Wide Association Studies (GWAS)  have linked a large number of genetic variants to psoriasis and other complex diseases. These variants however do not directly affect the coding region of genes but instead often lie in regulatory regions such as enhancers. These can that act through long range mechanisms and it may therefore be difficult to assign them to a specific gene. Moreover, regulatory regions are highly dynamic and context specific, requiring us to study them in disease relevant tissues and samples.</p>
<p>The objective in our lab is to use functional genomics techniques to uncover the underlying  biological mechanisms that can contribute to disease risk.</p>
<p>In the figure below you can see an example of our general workflow:</p>
<ul>
<li>An example GWAS loci contains multiple SNPs far away from genes making it difficult understanding their role in disease.</li>
<li>We apply a combination of techniques such as ChIP-seq, eQTL and chromatin conformation.</li>
<li>This results in a better understanding of the functional aspects underlying this loci such as identifying which genes are affected.</li>
</ul>
<p><img src="images/figure_1.png" alt="chromatin looping figure"></p>

                </section>
            </article>
            <div class='post-after'></div>
        </div>
    
        <div class='post-holder'>
            <article id='this-project' class='post '>
                <header class="post-header">
                    <h2 class="post-title" style="display:none">This project</h2>
                </header>
                <section class="post-content">
                    <h5 id="gwas-snps-affect-elements-that-are-highly-cell-type-specific">GWAS SNPs affect elements that are highly cell type specific</h5>
<p>The majority of GWAS SNPs has been associated with enhancer regions that are highly cell type and condition specific. This requires the study of these loci in cell types that are relevant to the traits studied.</p>
<p>For example dermtalogical conditions such as psoriasis are known to enrich in cell types such as keratinocytes.</p>
<p><img src="images/SNPs_enrichment.png" alt="SNP enrichment figure"></p>
<p>Despite this most previous studies have focused on immune cell types derived from blood as these are simpler to isolate and study.</p>
<p><strong>Here we used HiChIP, Hi-C and RNA-seq to study the global chromatin conformation and activity in cells derived from Human Keratinocytes (HaCaT) and a CD8 T cell line derived from a cancerous skin-plaque (MyLa).</strong></p>
<p><img src="images/graphical_abstract.png" alt="Graphical Abstract"></p>

                </section>
            </article>
            <div class='post-after'></div>
        </div>
    
        <div class='post-holder'>
            <article id='pathways' class='post '>
                <header class="post-header">
                    <h2 class="post-title" style="display:none">Pathways</h2>
                </header>
                <section class="post-content">
                    <p>We used our datasets to link genes to disease associated loci from 5 dermatological conditions. This results in between 84 and 399 potentially linked genes identified for each disease studied. Below you can see that these genes strongly enriched for disease relevant pathways.</p>
<p><img src="images/pathways.png" alt="pathways figure"></p>

                </section>
            </article>
            <div class='post-after'></div>
        </div>
    
        <div class='post-holder'>
            <article id='ets1' class='post '>
                <header class="post-header">
                    <h2 class="post-title" style="display:none">ETS1</h2>
                </header>
                <section class="post-content">
                    <h5 id="using-these-datasets-to-investigate-the-underlying-mechanisms-in-disease-associated-loci">Using these datasets to investigate the underlying mechanisms in disease associated loci</h5>
<p>We first looked at the ETS1 locus in particular. Here psoriasis and atopic dermatitis have distinct associations. However, whilst the psoriasis associated variants overlap the promoter of ETS1 directly, the association with atopic dermatitis is located 130 kb downstream of the gene.</p>
<p>Using our HiChIP interactions we can link the atopic dermatitis associated variants with the ETS1 promoter. This suggests a putative mechanism in which these two disting associations are mediated by a single gene.</p>
<p><img src="images/ETS1.png" alt="ETS1 figure"></p>

                </section>
            </article>
            <div class='post-after'></div>
        </div>
    
        <div class='post-holder'>
            <article id='satb1' class='post '>
                <header class="post-header">
                    <h2 class="post-title" style="display:none">SATB1</h2>
                </header>
                <section class="post-content">
                    <h5 id="linking-genes-with-psoriasis-loci">Linking genes with psoriasis loci</h5>
<p>We then focused on 4 psoriasis loci which show previously unknown candidate genes.</p>
<p>The first one is the SATB1 locus. Here we link the psoriasis SNP rs73178598 to SATB1 which is located 240kb away.
This gene is an important regulator of regulatory T cells and autoimmunity.</p>
<p><img src="images/SATB1.png" alt="SATB1 figure"></p>

                </section>
            </article>
            <div class='post-after'></div>
        </div>
    
        <div class='post-holder'>
            <article id='irf4' class='post '>
                <header class="post-header">
                    <h2 class="post-title" style="display:none">IRF4</h2>
                </header>
                <section class="post-content">
                    <p>The Ps loci indexed by SNP rs9504361 is intronic to EXOC2 and is typically associated with this gene. Here we show long-range interactions between this SNP and the promoters of IRF4 and DUSP22.</p>
<p><img src="images/IRF4.png" alt="IRF4 figure"></p>

                </section>
            </article>
            <div class='post-after'></div>
        </div>
    
        <div class='post-holder'>
            <article id='grhl3' class='post '>
                <header class="post-header">
                    <h2 class="post-title" style="display:none">GRHL3</h2>
                </header>
                <section class="post-content">
                    <p>Another example of a novel gene target can be found at the 1p36 locus (rs10794648), which to-date has been associated with IFNLR1. However, our HiChIP data showed long-range interactions between the Ps SNPs at 1p36 and the distal gene GRHL3 primarily in HaCaT keratinocytes. GRHL3 encodes a transcription factor which stimulates keratinocyte migration and has previously been independently linked to psoriasis.</p>
<p><img src="images/GRHL3.png" alt="GRHL3 figure"></p>

                </section>
            </article>
            <div class='post-after'></div>
        </div>
    
        <div class='post-holder'>
            <article id='foxo1' class='post '>
                <header class="post-header">
                    <h2 class="post-title" style="display:none">FOXO1</h2>
                </header>
                <section class="post-content">
                    <p>Finally, the locus indexed by the Ps SNP rs73183592 with previously undescribed mechanism was linked via a long-range interaction spanning about 500 kb to FOXO1 a gene with important functions in regulatory T cells.</p>
<p><img src="images/FOXO1.png" alt="FOXO1 figure"></p>

                </section>
            </article>
            <div class='post-after'></div>
        </div>
    
        <div class='post-holder'>
            <article id='conclusion' class='post '>
                <header class="post-header">
                    <h2 class="post-title" style="display:none">Conclusion</h2>
                </header>
                <section class="post-content">
                    <h2 id="conclusion">Conclusion</h2>
<p>Chromatin conformation and functional genomics studies have the potential to uncover the underlying mechanisms that drive the disease susceptibility of many complex traits. Here we used H3K27ac HiChIP, a novel technique that allows combined analysis of both chromatin conformation and chromatin activity, to create the first global study of promoter-enhancer interactions in keratinocyte and CD8+ T cell lines.</p>
<p>We use this data to explore disease associated SNPs for Psoriatic Arthritis, Psoriasis, Atopic Dermatitis, Melanoma and Systemic Sclerosis, and identify all the genes that are linked by chromatin interactions to these variants. All of these results are available on the bioRxiv preprint and further updates will come when our article is published.</p>
<p>The HiChIP gene targets have the potential to be used as therapeutic targets in drug repurposing and discovery, as recently applied for other diseases. However, whilst the techniques used in this study are an important strategy to identify potential candidate genes, they require further individual experimental validation (such as genetic perturbation techniques) before they can be used for drug targeting.</p>
<p>Regardless, we hope that these genes and mechanisms identified here represent a useful resource for further research aimed at characterising how genetic variation impacts on disease susceptibility for these and other complex diseases.</p>
<p><strong>Thank you for reading!</strong></p>

                </section>
            </article>
            <div class='post-after'></div>
        </div>
    
        <div class='post-holder'>
            <article id='contact' class='post last'>
                <header class="post-header">
                    <h2 class="post-title" style="display:none">Contact</h2>
                </header>
                <section class="post-content">
                    <p>This website was created by Chenfu Shi to display the poster presented at the Wellcome Virtual Conference &ldquo;Epigenomics of Common Diseases&rdquo; 2020.</p>
<p>For more information contact:
<i class="fa fa-envelope"></i>
 <a href="mailto:chenfu.shi@postgrad.manchester.ac.uk">chenfu.shi@postgrad.manchester.ac.uk</a></p>

                </section>
            </article>
            <div class='post-after'></div>
        </div>
    
</main>
<footer class="site-footer">
  <div class="inner">
    
      <section class="copyright">© Chenfu Shi</section>


    <section></section>
    
      <section>
        <a href="https://themes.gohugo.io/hugo-scroll/">Design</a> template
        built with ♥️ by
        <a href="https://www.janraasch.com" title="Jan Raasch">Jan Raasch</a>
      </section>

  </div>
</footer>

    <script
      type="text/javascript"
      src="https://code.jquery.com/jquery-1.11.3.min.js"
    ></script>

    
    <script type="text/javascript" src='js/icons.js'></script>
    <script type="text/javascript" src='js/index.js'></script>
    
</body>
</html>
